app/Http/Controllers/OverviewController.php

```diff
@@ -104,7 +105,15 @@ public function default(Request $request)
         $hide_dashboard_editor = UserPref::getPref($user, 'hide_dashboard_editor');
         $widgets               = Widget::select('widget_id', 'widget_title')->orderBy('widget_title')->get();
 
-        return view('overview.default', compact('bare', 'dash_config', 'dashboard', 'hide_dashboard_editor', 'user_dashboards', 'shared_dashboards', 'widgets'));
+        $user_list = User::select(['username', 'user_id'])
+            ->where('user_id', '!=', $user->user_id)
+            ->orderBy('username')
+            ->get()
+            ->filter(function ($users) use ($user) {
+                return $user->isAdmin();
+            });
```

Comment:
Now you get all users from database, then loop the array and filter if the user is admin.
```suggestion
    if ($user->can('manage', User::class) {
        $user_list = User::select(['username', 'user_id'])
            ->where('user_id', '!=', $user->user_id)
            ->orderBy('username')
            ->get();
    }
```